isis/src/galileo/objs/SsiCamera/SsiCamera.cpp

```diff
@@ -81,7 +81,7 @@ namespace Isis {
     // Get the start time in et
     PvlGroup inst = lab.findGroup("Instrument", Pvl::Traverse);
 
-    double et = iTime((QString)inst["StartTime"]).Et();
+    double et = getClockTime((QString)inst["SpacecraftClockStartCount"]).Et();
```

Comment:
Is this still necessary when using setImage call to get sun distance?

Reply:
Good catch, thanks! I'll move it into the catch.